app/controllers/greetings_controller.rb

```diff
@@ -1,9 +1,8 @@
 class GreetingsController < ApplicationController
-  before_action :set_greeting, only: %i[ show edit update destroy ]
+  before_action :set_greeting, only: %i[show edit update destroy]
 
   # GET /greetings or /greetings.json
   def index
     @greetings = Greeting.all
```

Comment:
I am unable to see a random greeting. This issue was pointed out by the previous reviewer as well. Kindly fix the issue before asking for a review.

Reply:
Thank you for your review. Please run `rails db:seed`.